src/app/template-library/template-library.component.scss

```diff
@@ -0,0 +1,85 @@
+.category-card-container {
```

Comment:
There are redundant classes here.
They have been moved to it's own component, so no need to leave them here anymore